src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import duke.command.Command;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+/**
+ * <h1>Duke Task Manager</h1>
+ * The Duke program a that the creation of tasks such as Todo, Event and
+ * Deadline. The program allows to listing of those added tasks and mark them as
+ * done and not done, it also allows deletion of wrongly added task.
+ *
+ * @author Leng Wei Cong, Justin
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+    * Constructor.
```

Comment:
Shouldn't there be a space between your comment header description and the parameter section? I think it applied for all comment headers.